images/build-libs/Makefile

```diff
@@ -0,0 +1,47 @@
+SHELL := /usr/bin/env bash
+
+cosign := $(shell command -v cosign 2>/dev/null)
+cosign_version ?= 1.9.0
+cosign_key ?= k8s://cosign/openinfra-oci-sign
+
+IMG_SLUG := test-infra
+IMG_NAME := build-libs
+IMG_TAG ?= latest
+
+ACCOUNT := 292999226676
+DOCKER_PUSH_REPOSITORY = dkr.ecr.eu-west-1.amazonaws.com
+
+IMAGE := "$(ACCOUNT).$(DOCKER_PUSH_REPOSITORY)/$(IMG_SLUG)/$(IMG_NAME):$(IMG_TAG)"
+
+ARCHS := "arm64,amd64"
+
+# To avoid using 'docker manifest' (that is experimental), 
+# allow buildx to manage push to registry for us.
+# buildx manages build and push in one step
+build-push: push-image
+
+create-builder:
+	# Workaround for buildx issue, see https://github.com/docker/buildx/issues/495#issuecomment-761562905
+	docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+	docker buildx create --name libsbuilder --driver docker-container --use
```

Comment:
I think you could use "--bootstrap" flag in here, no need an extra command for this.

Reply:
Thank you @developer-guy! I think this image (`build-libs`) image and related stuff can be removed, now that the CI has been moved to Github Actions.